monai/transforms/croppad/array.py

```diff
@@ -300,70 +298,146 @@ class DivisiblePad(Transform):
     def __init__(
         self,
         k: Union[Sequence[int], int],
-        mode: Union[NumpyPadMode, PytorchPadMode, str] = NumpyPadMode.CONSTANT,
         method: Union[Method, str] = Method.SYMMETRIC,
+        mode: str = PytorchPadMode.CONSTANT,
```

Comment:
the position of `method` and `mode` are swapped, perhaps we keep the original ones to avoid breaking change.

Reply:
Sure, restored in the latest commit.

Thanks.